services/openshift/scripts/openshift_provision

```diff
@@ -112,6 +112,55 @@ if [ ! -f ${ORIGIN_DIR}/secured.registry ]; then
   oc get route docker-registry -o json  | sed -e 's/\("spec": {\)/\1 "tls": {"termination": "passthrough"},/g' | oc replace -f -
 fi
 
+
+# Provision persistence volumes
+if [ ! -f ${ORIGIN_DIR}/configured.nfs ]; then
+  echo "[INFO] Creating and configuring NFS"
+  mkdir -p /nfsvolumes/pv{01..03}
+  chown -R nfsnobody:nfsnobody /nfsvolumes
+  chmod -R 777 /nfsvolumes
+  echo '' > /etc/exports
+
+  for i in {01..03}
+  do
+    echo "/nfsvolumes/pv${i} *(rw,root_squash)" >> /etc/exports
+  done
+  # To allow pods to write to remote NFS servers
+  setsebool -P virt_use_nfs 1
+
+  # Start and enable nfs
+  systemctl start rpcbind nfs-server
```

Comment:
@praveenkumar  I don't think you should start/enable rpcbind service since NFSv4 which openshift requires and which you are using AFAIK. But I am no expert in this area...
OpenShift requires NFSv4 when using NFS for persistent storage, due to NSFv4’s ability to handle SELinux labeling. [1]
NFSv4 works through firewalls and on the Internet, no longer requires an rpcbind service, ... [2]

[1] https://docs.openshift.com/enterprise/3.0/admin_guide/persistent_storage_nfs.html
[2] https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/Storage_Administration_Guide/ch-nfs.html